.travis.yml

```diff
@@ -23,26 +23,32 @@ matrix:
         - echo $JVM_EXTRA_ARGS
         - echo $JDK_PROFILES
       script: mvn package -Pall-java,licenses,lgpl,checkstyle,examples $JDK_PROFILES -DskipTests -Dmaven.javadoc.skip=true -B -V
-      
-      jobs:
-        include:
-          - jdk: openjdk8
-            env:
-            - JDK_PROFILES='-Pall-scala,scala'
-          - jdk: openjdk11
-            # ignite-scalar module currently doesn't build under jdk11
-            env:
-              - JDK_PROFILES='-P!spark-2.4,!spark,!scala'
-            before_script:
-            - |
-              export JVM_EXTRA_ARGS='--illegal-access=permit
-              --add-exports=java.base/jdk.internal.misc=ALL-UNNAMED
-              --add-exports=java.base/sun.nio.ch=ALL-UNNAMED
-              --add-exports=java.management/com.sun.jmx.mbeanserver=ALL-UNNAMED
-              --add-exports=jdk.internal.jvmstat/sun.jvmstat.monitor=ALL-UNNAMED
-              --add-exports=java.base/sun.reflect.generics.reflectiveObjects=ALL-UNNAMED
-              -Djdk.tls.client.protocols=TLSv1.2'
-      
+      jdk: openjdk8
+      env:
+        - JDK_PROFILES='-Pall-scala,scala'
+      before_cache: rm -rfv $HOME/.m2/repository/org/apache/ignite
+      cache: {directories: [$HOME/.m2/repository]}
+
+    - language: java
+      os: linux
+      dist: trusty
+      install: skip
+      before_script:
+        - echo $JVM_EXTRA_ARGS
+        - echo $JDK_PROFILES
+        - |
+          export JVM_EXTRA_ARGS='--illegal-access=permit
+          --add-exports=java.base/jdk.internal.misc=ALL-UNNAMED
+          --add-exports=java.base/sun.nio.ch=ALL-UNNAMED
+          --add-exports=java.management/com.sun.jmx.mbeanserver=ALL-UNNAMED
+          --add-exports=jdk.internal.jvmstat/sun.jvmstat.monitor=ALL-UNNAMED
+          --add-exports=java.base/sun.reflect.generics.reflectiveObjects=ALL-UNNAMED
+          -Djdk.tls.client.protocols=TLSv1.2'
+      script: mvn package -Pall-java,licenses,lgpl,checkstyle,examples $JDK_PROFILES -DskipTests -Dmaven.javadoc.skip=true -B -V
+      jdk: openjdk11
+      # ignite-scalar module currently doesn't build under jdk11
+      env:
+        - JDK_PROFILES='-P!spark-2.4,!spark,!scala'
```

Comment:
I think `JDK_PROFILES` is not needed anymore since it can be directly set to an appropriate JDK profile.

Reply:
@Mmuzaf do you mean I should inline this env var, put the profiles directly to `script`?

Reply:
@ptupitsyn yes, exactly